routemaster/db/model.py

```diff
@@ -105,45 +105,44 @@
     'state_machines',
     metadata,
 
-    Column('name', String, primary_key=True),
-    Column('updated', DateTime),
+    Column('id', Integer, primary_key=True, autoincrement=True),
+    Column('name', String),
```

Comment:
While I'm not sure what model we're after, I wonder whether this column should be unique?

Reply:
Not for what James and I were trying to get to when we came up with this iteration. Old state machines would not be resurrected, a new one would be created.